app/uk/gov/hmrc/agentepayeregistration/models/RegistrationExtraction.scala

```diff
@@ -40,9 +40,9 @@ object RegistrationExtraction {
     RegistrationExtraction(agentReference.value,
       agentName,
       contactName,
-      telephoneNumber,
```

Comment:
I think the change of `telephoneNumber` to `phoneNo` and `emailAddress` to email here now makes the extraction API endpoint different to that described in the README ( `GET /agent-epaye-registration/registrations?dateFrom=yyyy-MM-dd&dateTo=yyyy-MM-dd` )

Reply:
Original story with the field names: APB-1125

Reply:
I think i commited something about half an hour ago changing these back to telephoneNumber and email because of what Artur said to me yesterday afternoon